blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -37,6 +37,10 @@ Unreleased Changes
 Breaking Changes
 ================
 
+- Byte size and time setting values do not require a unit suffix
```

Comment:
I thought this is the other way around? That we now support adding the units?

master:

```
create table t (x int) with (refresh_interval = '5m');
SQLActionException[SQLParseException: Invalid value for argument 'refresh_interval']
```

this branch:

```
cr> create table t (x int) with (refresh_interval = '5m');
CREATE OK, 1 row affected  (0.221 sec)
```

Reply:
@mfussenegger right, statement parameters did not support it while configuration settings forced it.
pushed a fixup.